biowasm.json

```diff
@@ -399,6 +399,17 @@
 					"branch": "a12da33"
 				}
 			]
+		},
+		{
+			"name": "muscle",
+			"programs": ["muscle"],
+			"description": "MUSCLE is widely-used software for making multiple alignments of biological sequences",
+			"versions": [
+				{
+					"version": "5.1.0",
+					"branch": "v5.1.0"
```

Comment:
This branch doesn't exist. This should be `5.1.0`: https://github.com/rcedgar/muscle/tree/5.1.0

Reply:
thanks！ I will upload these missing  files after a few days~

Reply:
@robertaboukhalil I just upload the ```update``` commit, please help me check if everything is right, thanks~